.github/workflows/pre-commit.yaml

```diff
@@ -16,4 +16,5 @@ jobs:
         python -m ensurepip
         python -m pip install --upgrade pip
         python -m pip install .[dev]
+        npm install --prefix dtbase/webapp/ --include=dev
```

Comment:
Is this only needed for the `eslint-plugin-prettier` thing, or does something else require this too? If it's only the plugin, do we really need it?